shell/common/gin_converters/content_converter.cc

```diff
@@ -228,6 +228,8 @@ v8::Local<v8::Value> Converter<content::PermissionType>::ToV8(
       return StringToV8(isolate, "system-wake-lock");
     case content::PermissionType::WINDOW_PLACEMENT:
       return StringToV8(isolate, "window-placement");
+    case content::PermissionType::DISPLAY_CAPTURE:
+      return StringToV8(isolate, "display-capture");
```

Comment:
Let's make an issue for this and add it as a new semver-minor PR as noted above

Reply:
Issue created! I left in the above API change, but removed the documentation. We can add that into the semver-minor: https://github.com/electron/electron/issues/27576